packages/mobile/components/drawers/DrawerManager.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+    import { localize } from '@core/i18n'
+    import { Drawer } from '../../components'
+    import { closeDrawer, DrawerId, drawersStore } from '../../lib/auxiliary/drawer'
+    import ConfirmDrawer from './ConfirmDrawer.svelte'
+    import EnterPasswordDrawer from './EnterPasswordDrawer.svelte'
+    import SelectedActivityDrawer from './SelectedActivityDrawer.svelte'
+
+    const COMPONENTS = {
+        [DrawerId.Confirm]: ConfirmDrawer,
+        [DrawerId.EnterPassword]: EnterPasswordDrawer,
+        [DrawerId.SelectedActivity]: SelectedActivityDrawer,
+    }
```

Comment:
Maybe we create a constant file for this, so we do not have to touch the manager every time we add a drawer?